### PR TITLE
FIPS mode detection

### PR DESCRIPTION
**Pull request by acardace**
**What this PR does / why we need it**:
This PR logs a message in all controllers when the container is running in FIPS mode which more specifically means that `/proc/sys/crypto/fips_enabled` is `1` and the installed OpenSSL library is FIPS-capable.

The log message does not mean that KubeVirt itself is running in FIPS mode.

The purpose of this is debuggability in order to quickly be able to inspect whether a container is correctly running in FIPS mode.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/fips/fips.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+			if validLibName.MatchString(entries[i].Name()) {
+				libs = append(libs, entries[i].Name())
+			}
+		}
+	}
+	return libs
+}
+
+func findCryptoLibs() ([]string, error) {
+	libPaths := []string{"/lib64", "/usr/lib64", "/lib", "/usr/lib"}
```
Comment (stu-gott) [inline pkg/util/fips/fips.go:73]: Should this list of paths be placed at the top of the module?

**Review by stu-gott — COMMENTED** [review]
Looks great! minor nit only.

**Comment by rmohr** [comment]
/hold

As discussed on the last PR and on slack I am not convinced that we should add something like this. I want to discuss this with other maintainers first.

File: pkg/util/fips/fips.go (hunk 1 above)
Comment (acardace) [reply pkg/util/fips/fips.go]: moved it at the top. Thanks!

**Pull request closed without merging**